src/components/CheckBox.tsx

```diff
@@ -1,5 +1,9 @@
 import React, { Dispatch, ReactNode, SetStateAction } from "react";
 
+/**
+ * Renders a checkbox
+ * @group Components
```

Comment:
I'm not familiar with typedocs, what does `@group Components` do?

Reply:
TypeDoc appears to be more oriented towards documenting libraries, and doesn't naturally support the structural paradigm of the application code. `@group Components` seeks to overcome this limitation by grouping the React Components under the same heading in the generated docs.

Reply:
Have you used typedoc with react before? If it’s not meant for it I’m wondering if it’s worth doing versus adding more comments / refactoring the code.

I’m not saying this won’t work or I won’t approve it, but I just want to make sure this is thought out and beneficial

Reply:
@thompsondt I'll let you decide, if you think this helps new contributors then I think that's great